_studio/mfx_lib/vpp/src/mfx_vpp_sw_core.cpp

```diff
@@ -426,8 +426,27 @@ mfxStatus VideoVPPBase::QueryIOSurf(VideoCORE* core, mfxVideoParam *par, mfxFram
 
         mfxSts = CheckIOPattern_AndSetIOMemTypes(par->IOPattern, &(request[VPP_IN].Type), &(request[VPP_OUT].Type), bSWLib);
         MFX_CHECK_STS(mfxSts);
-        return (bSWLib)? MFX_ERR_UNSUPPORTED : MFX_ERR_NONE;
+        return (bSWLib)? MFX_WRN_PARTIAL_ACCELERATION : MFX_ERR_NONE;
     }
```

Comment:
We have neither SW lib nor SW fallbacks in open source, so unsupported should be returned here.

Reply:
MCTF is not HW implementation so bSWLib is set to true.
In close source, when camera plugin is not requested we have:
return (bSWLib)? MFX_WRN_PARTIAL_ACCELERATION : MFX_ERR_NONE; 

Are you suggesting that we do not follow the logic of close source to support MCTF ?
Feel free to provide your own suggestion.